### ISSUE TITLE
lodash.round(-1.005, 2) results in -1

### ISSUE THREAD

**Issue by brainz80** [lodash/lodash#5907]
There seems to be a bug with how negative numbers are rounded.

The solution used for rounding numbers in Lodash seems pretty good for positive numbers, but when e.g. trying to round `-1.005` to 2 decimals the result is `-1` when it should be `-1.01`

A solution like this should fix this issue:

```js
function round (number, precision = 2) {
    return Math.sign(number) * Number(`${Math.round(`${Math.abs(number)}e+${precision}`)}e-${precision}`);
}
```

Here I get the absolute of `number` instead of the actual number and after doing the roundings I multiply the result with the sign (`-1` or `1`) of `number`

**Comment by saranchonkau** [comment]
I guess current behaviour is correct. You may try to execute `Math.round(-100.5)`, and get result `-100`, but not `-101`.

**Comment by shameen** [comment]
"Rounding half up" is the default rounding method in JavaScript, so `Math.round(-0.5)` returns `0`

It sounds like you want "Round half away from zero", which could be implemented like
```js
/**
 * Rounds a number, where midpoint values are rounded away from zero
 * @example
 * roundHalfAwayFromZero(0.5)   // 1
 * roundHalfAwayFromZero(-0.5)  // -1
 */
const roundHalfAwayFromZero = (num, places = 0) => {
  return num < 0 ? -_.round(-num, places) : _.round(num, places);
};
```

**Comment by ljharb** [comment]
Indeed, rounding goes up, not away from zero, so this is correct as is.
